bb8-utils-gitlab.env

```diff
@@ -715,7 +722,7 @@ _t4dDebugInfoStart "_t4dModBb8UtilsGet" "$#" "$@"
 #
 ###
 #   - Example:
-#       _t4dModBb8UtilsGet https://<server>/<key>/<project>/-/merge_requests/<id>
+#       _t4dModBb8UtilsGet https://<server>/<key>/<project>/-/merge_requests/<id>>
```

Comment:
humm this is strange.

Reply:
Added because it gets eaten in the generated md for some reason

Reply:
humm yes hard to find a regex in md generator, but i don't know if adding a workaround like this is a good idea ^^

Reply:
i think i fix this by using `{id}`. 

<> values are for usage purpose or highlights in terminal, basically surrounded by spaces not inside urls.